_install/synology.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: documentation
+title:  Synology NAS hosting
+date:   2021-01-10
+order:  99
+---
+
+This procedure if for Synology NAS running DiskStation Manager (DSM) version 6.X.
+
+# Pre-requisites
+
+## Packages Installation
+
+Open the DiskStation Manager interface. Use the Package Center to install the following packages (note they may already be installed):
+* Web Station
+* MariaDB 10
+* phpMyAdmin (only required for initial set-up of the database, may be removed once Webtrees is running).
+
+## Webstation configuration
+
+1. In DSM, open "Web Station".
+2. Click on "General Setting" and check which PHP profile is being used ("Default PHP 7.4" in the below example):
+
+   ![Click on General Settings, check PHP profile.](../assets/img/synology/WS-GeneralSettings.png)
+
+3. Click on "PHP Settings" then select the the PHP profile used by Web Station:
+
+   ![Click on PHP Settings, select default PHP profile.](../assets/img/synology/WS-PHPSettings.png)
+
+4. Edit the selected PHP profile, make sure all extensions are activated in the Extensions list. Click OK:
+
+   ![Edit the default PHP profile, activate all extensions.](../assets/img/synology/WS-PHPProfileSettings.png)
+
+## MySQL/MariaDB configuration
+
+### Confirm MySQL/MariaDB server port
+
+5. In DSM, open "MariaDB 10".
+6. Check/confirm which port MariaDB is running on. Default is 3307.
+
+   ![Check the MariaDB 10 port. Default is 3307 on Synology NAS.](../assets/img/synology/WS-MariaDBSettings.png)
+
+### MariaDB user and database configuration
+
+7. In DSM, open "phpMyAdmin", and log-in using a profile allowed to create databases and profiles/users (e.g. "root").
+8. Click on "User account", then click on "Add user account":
+
+   ![Add a user account.](../assets/img/synology/WS-MariaUserAccounts.png)
+
+9. Choose the database sessings:
+- "User name" will be both the user name and the database name. **Note it in order not to forget it!**
+- "Host name" must be set to "localhost" for safety reasons.
+- Generating a complex/random password is recommended. **Note it in order not to forget it!**
+- Tick "Create database with same name and grant all privileges."
+
+   ![Recommended user settings.](../assets/img/synology/WS-MariaUserAccountsSettings.png)
+
+10. Click "Go" at the bottom right to generate both the user and its associated database.
+
+# Webtrees installation
+
+## Moving the files to your NAS
+
+11. Synology NAS come with PHP >= 7.1, so you will have to download [webtrees-{{ site.latest_version }}.zip][DOWNLOAD].
```

Comment:
We are just about to drop support for PHP 7.1 and PHP 7.2

The next major update (2.1) will require PHP 7.3.

Reply:
OK, will edit the doc to show how to make sure the highest version of PHP available on Synology NAS (7.4) is installed (and how to set it as the default).